src/rustdoc_schema.graphql

```diff
@@ -415,6 +415,8 @@ type Trait implements Item & Importable {
   method: [Method!]
 
   supertrait: [ImplementedTrait!]
+
+  associated_type: [AssociatedType!]!
```

Comment:
```suggestion
  associated_type: [AssociatedType!]
```
because not all traits have associated types

Reply:
this is actually kind of confusing, the way `[AssociatedType!]!` reads to me is iterator of AssociatedType, the way `[AssociatedType!]` reads to me is iterator or null of AssociatedType, which doesn't really even make sense to me, would you be able to clarify how I should interpret this change?

Reply:
I made this change regardless of that^ in a6e05ff

Reply:
Oh okay, that makes perfect sense